website/docs/reference/widgets/custom.md

```diff
@@ -352,12 +352,20 @@ appsmith.onReady(() => {
 
 <dd>
 
-Subscribe to theme changes and execute a callback function.
+The `onThemeChange` function allows you to subscribe to theme changes within the application and execute a callback when the theme is updated. This facilitates the dynamic adjustment of UI elements to reflect the current theme. The following theme properties are accessible:
```

Comment:
Add details for how methods are used inside the code snippet. Simplify the language to say that the theme changes from application will be propagated to custom widget using the onThemeChange method.